tests/tools/test_release_evidence_snapshot.py

```
@@ -0,0 +1,281 @@
+
+def test_release_evidence_snapshot_catalog_checksum_mismatch_fails_closed(
+    tmp_path: Path, monkeypatch: Any
+) -> None:
+    """Artifact catalogs with stale checksums should not pass the release evidence gate."""
+    repo = _init_fixture_repo(tmp_path / "repo")
+    monkeypatch.chdir(repo)
+    table_path = repo / "docs/context/evidence/issue_fixture/tables/campaign_table.md"
+    table_path.write_text(
+        "| planner | status |\n| --- | --- |\n| orca | degraded |\n", encoding="utf-8"
+    )
+    subprocess.run(["git", "add", "."], cwd=repo, check=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)